ports/esp32/network_ppp.c

```diff
@@ -250,7 +254,11 @@ STATIC mp_obj_t ppp_ifconfig(size_t n_args, const mp_obj_t *args) {
         ip_addr_t dns;
         mp_obj_t *items;
         mp_obj_get_array_fixed_n(args[1], 4, &items);
+        #if CONFIG_LWIP_IPV6
         netutils_parse_ipv4_addr(items[3], (uint8_t *)&dns.u_addr.ip4, NETUTILS_BIG);
+        #else
+        netutils_parse_ipv4_addr(items[3], (uint8_t *)&dns.addr, NETUTILS_BIG);
```

Comment:
I think this should be `(uint8_t *)&dns`, because `dns.u_addr.ip4` is the same type as `dns` when IPV6 is disabled.

Reply:
It should still work, as `struct ip4_addr` has only one member.  However, I noticed `.addr` wasn't used anywhere else so I made the change you suggested to keep consistency.

Reply:
Thanks for updating.